common-design/components/cd-navigation.html

```diff
@@ -10,13 +10,13 @@
   <nav role="navigation" id="cd-nav" class="cd-nav cd-dropdown" aria-labelledby="cd-nav-toggle">
     <ul>
       <li class="cd-nav__item">
-        <button type="button" id="cd-sub-nav-toggle" data-toggle="dropdown">
+        <button type="button" class="cd-dropdown-toggle cd-sub-nav-toggle" data-toggle="dropdown">
```

Comment:
Change the ID to a class.
Drupal won't have button elements as the first item in the list
And it will have multiple dropdown menus
Drupal adds a class to any li that has a nested ul
This allows for closer matching of the JS selectors for the click event that displays the dropdown